independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/CreateProject.java

```diff
@@ -67,6 +68,11 @@ public CreateProject version(String version) {
         return this;
     }
 
+    public CreateProject parentProject(String parentPath) {
+        values.put(PROJECT_PARENT_PATH.key(), parentPath);
+        return this;
+    }
+
```

Comment:
The CLI doesn't use CreateProject .. just as a heads up. It bypasses this indirection (as another thing creating a map before you create the command invocation which is a map.. )

Reply:
will have a better look at this later, Im quite unaware of the current state of cli/maven/codestarts

Reply:
I guess you can keep it for now @cristian-com. 

@ebullient maybe we should remove CreateProject, if it's not the recommended way of doing things?

Reply:
I think the maven plugin still uses it.

I was merging the jbang and maven/gradle project create paths, and CreateProject didn't allow what I wanted to do for a prepare step that would allow a meaningful dryrun.

I would like to disentangle a lot of that code.